react-charts/donut-utilization-chart/step5.md

```diff
@@ -1,32 +1,56 @@
-Now we'll add tooltips to the donut utilization chart...
+Add a legend to the donut utilization chart.
 
-## Task
+1) <strong>Make sure the App.js file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the code for the `<Chart>` component.</strong>
 
-2) Locate the code that looks like the following:
+It should look like this:
 
 <pre class="file">
 &lt;ChartDonutUtilization
+  constrainToVisibleArea
   data={{ x: &#39;Storage capacity&#39;, y: 45 }}
   title=&quot;45%&quot;
   subTitle=&quot;of 100 GBps&quot;
+  labels={({ datum }) =&gt; datum.x ? `${datum.x}: ${datum.y}%` : null}
   thresholds={[{ value: 60 }, { value: 90 }]}
   height={230}
   width={350}
 /&gt;
 </pre>
 
-3) Add the following properties to that section:
+3) <strong>Add a `padding` property to the component.</strong>

```

Comment:
Follow what is in area-chart:

<img width="334" alt="Screen Shot 2019-12-05 at 10 46 17 PM" src="https://user-images.githubusercontent.com/20118816/70294323-10feb680-17b1-11ea-9ecc-f0d6f9bb1ecf.png">

Reply:
@christiemolloy the code for the padding property is different in this module - do you want me to change those values to match the area-chart exactly or keep the values donut-chart currently has:
- bottom: 20
- left: 20
- right: 290
- top: 20

Reply:
got it - will keep the padding values as currently defined.